src/Autowhatever.js

```diff
@@ -281,7 +283,7 @@ export default class Autowhatever extends Component {
 
     return (
       <div {...theme(`react-autowhatever-${id}-container`, 'container', isOpen && 'containerOpen')}>
-        <input {...inputProps} />
+        {this.props.renderInput(inputProps)}
```

Comment:
Great! Could you please update the PR with `inputElement`, and add a demo? If you could add tests as well that would be great.